elasticache_auto_discovery/__init__.py

```diff
@@ -2,23 +2,20 @@
 
 import socket
 
-
 def discover(configuration_endpoint, time_to_timeout=None):
     host, port = configuration_endpoint.split(':')
-
     configs = []
     sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     if time_to_timeout is not None:
         sock.settimeout(time_to_timeout)
 
     try:
         sock.connect((host, int(port)))
-        sock.sendall('config get cluster\r\n')
-
+        sock.sendall(str.encode('config get cluster\r\n'))
         data = ''
```

Comment:
if you make this a bytestring to start with (`b''`), then you won't have to append unicode to it later